tests/modules/nf-core/gem2/gem2bedmappability/test.yml

```diff
@@ -0,0 +1,11 @@
+- name: gem2 gem2bedmappability test_gem2_gem2bedmappability
+  command: nextflow run ./tests/modules/nf-core/gem2/gem2bedmappability -entry test_gem2_gem2bedmappability -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/gem2/gem2bedmappability/nextflow.config
+  tags:
+    - gem2
+    - gem2/gem2bedmappability
+  files:
+    - path: output/gem2/test.bg
+      md5sum: 42596230d57c308892bcaefb3eabbf1b
+    - path: output/gem2/test.sizes
+      md5sum: 58102c0866e0bc69831c94d02fd859e1
+    - path: output/gem2/versions.yml
```

Comment:
Could you maybe add some lines of the output files?

Reply:
Why would I do that if the md5sum works? :p

Reply:
Fair point, just to see the output of the file :)

Reply:
`test.bg`:
```
MT192765.1 Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/USA/PC00101P/2020	0	29729	1
MT192765.1 Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/USA/PC00101P/2020	29729	29730	0
```

`test.sizes`:
```
MT192765.1 Severe acute respiratory syndrome coronavirus 2 isolate SARS-CoV-2/human/USA/PC00101P/2020	29829
```